editor/plugins/gradient_editor_plugin.h

```diff
@@ -33,25 +33,100 @@
 
 #include "editor/editor_inspector.h"
 #include "editor/editor_plugin.h"
-#include "gradient_editor.h"
+#include "scene/gui/color_picker.h"
+#include "scene/gui/popup.h"
```

Comment:
The other includes can be moved too 🙃
Includes in header are most often needed only if you inherit some class. Here only `editor_inspector.h` and `editor_plugin.h` are needed (they indirectly include other classes).